Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8
# [Core/Packet] SMSG_AUCTION_REPLICATE_RESPONSE implemented

* auction GetAll (a.k.a. full auction scan) works now.
* all addons that use `QueryAuctionItems("", nil, nil, 0, 0, 0, 0, 0, 0, true)` now function correctly.
* **TradeSkillMaster**'s `Run GetAll Scan` works now.

to test the PR:
* install TSM. https://github.com/3nei/TradeSkillMaster_MoP
* `Run GetAll Scan` in the TSM addon (in the AuctionDB tab).

or:
* install my addon. https://github.com/3nei/QueryAuctionItemsMoP
* open the auction and type in chat: `/fullscan`

Note:
Be aware that even if you set `Auction.GetAllScanDelay` and `Auction.SearchDelay` to `0`, the client will not allow you to scan more frequently than once every 15 minutes. The client has an internal cooldown on full scans. You need to log out and log back in to reset that full scan cooldown (assuming there is no server-side cooldown, of course). It appears that the scan is linked to a Unix timestamp, which the client updates after each scan to maintain a 15-minute gap.

https://youtu.be/ar2bGBiHLLA
